example/rcnn/README.md

```diff
@@ -43,8 +43,8 @@ Make a directory `data` and follow `py-faster-rcnn` for data preparation instruc
 * [MSCOCO](http://mscoco.org/dataset/) should be in `data/coco` containing `train2017`, `val2017` and `annotations/instances_train2017.json`, `annotations/instances_val2017.json`.
 
 ### Download pretrained ImageNet models
-* [VGG16](http://www.robots.ox.ac.uk/~vgg/research/very_deep/) should be at `model/vgg16-0000.params` from [MXNet model zoo](http://data.dmlc.ml/models/imagenet/vgg/).
-* [ResNet](https://github.com/tornadomeet/ResNet) should be at `model/resnet-101-0000.params` from [MXNet model zoo](http://data.dmlc.ml/models/imagenet/resnet/).
+* [VGG16](http://www.robots.ox.ac.uk/~vgg/research/very_deep/) should be at `model/vgg16-0000.params` from [MXNet model zoo](http://data.mxnet.io/models/imagenet/vgg/).
```

Comment:
This link doesn't work: http://data.mxnet.io/models/imagenet/vgg/
Looks like a redirect on s3 and without a specific key it won't work. And browsing is disabled... so... you'll need to go in on the s3 side to find out where this stuff is and get the full URL.

Reply:
Ok, use the AWS CLI to browse around:
```
aws s3 ls s3://data.mxnet.io
aws s3 ls s3://data.mxnet.io/mxnet/models/imagenet/vgg/
```
Then you should be able to figure out the links that way.

Reply:
Updated and verified the links